pkg/network/node/iptables.go

```diff
@@ -109,7 +140,11 @@ func (n *NodeIPTables) syncIPTableRules() error {
 			// Chain existed but not with the expected rules; this probably means
 			// it contained rules referring to a *different* subnet; flush them
 			// and try again.
-			if err = n.ipt.FlushChain(iptables.Table(chain.table), iptables.Chain(chain.name)); err != nil {
+			err = execIPTablesWithRetry(func() error {
+				err := n.ipt.FlushChain(iptables.Table(chain.table), iptables.Chain(chain.name))
```

Comment:
nit; can be `return n.ipt.FlushChain(iptables.Table(chain.table), iptables.Chain(chain.name))`

Reply:
done